client/blocks/daily-post-button/index.jsx

```diff
@@ -0,0 +1,181 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import page from 'page';
+import qs from 'qs';
+import { get, defer } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import { translate } from 'i18n-calypso';
+import { preload } from 'sections-preload';
+import SitesPopover from 'components/sites-popover';
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import { markSeen as markPostSeen } from 'lib/feed-post-store/actions';
+
+import getSitesList from 'lib/sites-list';
+import { recordGaEvent, recordAction, recordTrackForPost } from 'reader/stats';
+import { getDailyPostType } from './helper';
+
+const sitesList = getSitesList();
```

Comment:
this isn't a static value, it can / will change as sites-list loads and as users take actions across the site. We should treat it as a reactive thing. 

Need to investigate the state of the "move sites-list to redux" project and see if we can use a redux selector here, or if we have to bind to the sites-list emitter.